### PR TITLE
validator: explictly set trust roots

### PR DESCRIPTION
**Pull request by Freax13**
Currently, go-sev-guest doesn't have a root for Genoa, so we need to add it explicitly.

### CODE REVIEW

**Comment by katexochen** [comment]
We should open an upstream issue/pr to add the genoa cert upstream.

**Comment by Freax13** [comment]
> We should open an upstream issue/pr to add the genoa cert upstream.

https://github.com/google/go-sev-guest/pull/127

File: internal/attestation/snp/validator.go [hunk 1]
```diff
@@ -144,3 +146,32 @@ func (v *Validator) Validate(ctx context.Context, attDocRaw []byte, nonce []byte
...
+	milanCerts := trust.AMDRootCertsProduct("Milan")
+	err := milanCerts.FromKDSCertBytes(askArkMilanVcekBytes)
+	if err != nil {
+		panic(fmt.Errorf("failed to parse cert: %w", err))
+	}
+	trustedRoots["Milan"] = []*trust.AMDRootCerts{milanCerts}
+
+	genoaCerts := trust.AMDRootCertsProduct("Genoa")
+	err = genoaCerts.FromKDSCertBytes(askArkGenoaVcekBytes)
+	if err != nil {
```
Comment (katexochen) [inline internal/attestation/snp/validator.go:171]:
```suggestion
	
	if err := genoaCerts.FromKDSCertBytes(askArkGenoaVcekBytes); err != nil {
```

**Review by katexochen — COMMENTED** [review]
just some nits left.

**Review by katexochen — APPROVED** [review]

**Pull request merged**